docs/building-from-source.md

```diff
@@ -37,13 +37,13 @@ A good new-user tutorial for Visual Studio is available at [this](https://www.yo
 
 In the user's shell of choice, ensure that the Dotnet SDK is installed and available on the `PATH` for the shell. Navigate to the root of the repository, then execute the following command for building the repository:
 
-```
+```shell
 dotnet build
 ```
 
 Users can run the sbom tool using this command which contains the minimum required set of parameters:
 
-```
+```shell
```

Comment:
In this case, we want the < and > to be placeholders, not literals, so we want the generic formatting